coniferest/isoforest.py

```diff
@@ -78,6 +78,9 @@ def score_samples(self, samples):
     def fit_known(self, data, known_data=None, known_labels=None):
         return self.fit(data)
 
+    def pariou_transform(self, x, axis = None):
```

Comment:
Should we have it in a base class instead? And mark AADForest as not implemented?

Reply:
I make an abstract method in the base class. But we can't have its implementation in the base class, because the base class has no `evaluator` inside.

Reply:
Ah, I see. Technically we could, but of course we should not